docs/commerce/payment/configure_payment.md

```diff
@@ -0,0 +1,47 @@
+---
+description: Configure the payment process.
+edition: commerce
+---
+
+# Configure payment
+
+When you work with your Commerce implementation, you can review and modify the payment configuration.
+
+!!! note "Permissions" 
+
+    When you modify the workflow configuration, make sure you properly set user [permissions](permission_use_cases.md#commerce) for the Payment component.
+
+## Payment workflow
+
+Payment workflow relies on a [Symfony Workflow](http://symfony.com/doc/5.4/components/workflow.html).
+Each transition represents a separate payment step. 
+
+The default fallback workflow is `ibexa_payment`.
+
+### Default payment workflow configuration
+
+The default payment workflow configuration is called `ibexa_payment`, you can replace it with your custom workflow identifier if needed.
+
+To see the default workflow, in your project directory, navigate to the following file: `vendor/Ibexa/payment/src/bundle/Resources/config/workflow.yaml`.
```

Comment:
1) You can paste ibexa_payment workflow definition here. It will be more convenient for reader. 

2) You can add some description and even graph

Reply:
In general I'd avoid directly quoting code from the product, because it causes additional maintenance overhead (https://github.com/ezsystems/developer-documentation/pull/1966#discussion_r1174840530).